maps/graph_edges/base_graph/edges_nok.py

```diff
@@ -118,10 +118,10 @@
     {"from": {"map": "NOK_11", "id": 0}, "to": {"map": "MAC_01", "id": 1}, "reqs": []}, # Pleasant Path Entry Exit Left -> Plaza District Exit Right
     {"from": {"map": "NOK_11", "id": 1}, "to": {"map": "NOK_12", "id": 0}, "reqs": []}, # Pleasant Path Entry Exit Right -> Pleasant Path Bridge Exit Left
 
-    {"from": {"map": "NOK_11", "id": 0}, "to": {"map": "NOK_11", "id": 1}, "reqs": []}, #? Pleasant Path Entry Exit Left -> Pleasant Path Entry Exit Right
-    {"from": {"map": "NOK_11", "id": 1}, "to": {"map": "NOK_11", "id": 0}, "reqs": []}, #? Pleasant Path Entry Exit Right -> Pleasant Path Entry Exit Left
+    {"from": {"map": "NOK_11", "id": 0}, "to": {"map": "NOK_11", "id": 1}, "reqs": [["RF_CanPassKentC",{"starspirits": 2}],["RF_CanPassKentC","Goombario"],["RF_CanPassKentC","Boots"]]}, #? Pleasant Path Entry Exit Left -> Pleasant Path Entry Exit Right
```

Comment:
Maybe I'm reading it wrong, but doesn't this result in logically requiring 2 star spirits AND Goombario AND boots? According to the changelog it should be 2 star spirits AND (Goombario OR Boots)

Reply:
Yup, you're reading this right. The change log wasn't entirely accurate